docs/standard/garbagecollection/implementing-dispose.md

```diff
@@ -410,7 +410,10 @@ Classes derived from the [System.Runtime.InteropServices.SafeHandle](xref:System
 
 * The [SafeMemoryMappedViewHandle](xref:Microsoft.Win32.SafeHandles.SafeMemoryMappedViewHandle) class, for memory views. 
 
-* The [SafeNCryptKeyHandle](xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle), [SafeNCryptProviderHandle](xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle), and [SafeNCryptSecretHandle](xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle) classes, for cryptography constructs. 
+* The [SafeNCryptKeyHandle](https://msdn.microsoft.com/en-us/library/microsoft.win32.safehandles.safencryptkeyhandle(v=vs.110).aspx), [SafeNCryptProviderHandle](https://msdn.microsoft.com/en-us/library/microsoft.win32.safehandles.safencryptproviderhandle(v=vs.110).aspx), and [SafeNCryptSecretHandle](https://msdn.microsoft.com/en-us/library/microsoft.win32.safehandles.safencryptsecrethandle(v=vs.110).aspx) classes, for cryptography constructs. 
+
+  > [!Note]
```

Comment:
I forgot that I had added this note. Technically it's still accurate given that the APIs can't be accessed from core until the build issue is resolved. I can remove it now, or we can wait until the APIs come back and remove it when I restore the old links.